pkgs/applications/editors/vscode/extensions/default.nix

```diff
@@ -4133,6 +4133,21 @@ let
         };
       };
 
+      sainnhe.gruvbox-material = buildVscodeMarketplaceExtension {
+        mktplcRef = {
+          name = "gruvbox-material";
+          publisher = "sainnhe";
+          version = "6.5.2";
+          hash = "sha256-D+SZEQQwjZeuyENOYBJGn8tqS3cJiWbEkmEqhNRY/i4=";
+        };
+        meta = {
+          description = "Gruvbox with Material Palette";
```

Comment:
This doesn't say it's a VSCode extension. Please read the README in this directory.